packages/usa-tooltip/src/styles/_usa-tooltip.scss

```
@@ -26,6 +26,7 @@ $triangle-size: 5px;
   color: color($theme-tooltip-font-color);
   display: none;
   font-size: size("ui", $theme-tooltip-font-size);
+  opacity: 0;
```

We could add a note to avoid this in the future.

```suggestion
  opacity: 0; // Required for recalculating position.
```